.github/PULL_REQUEST_TEMPLATE/add_repo.md

```diff
@@ -0,0 +1,27 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+* Consider creating this PR as draft: https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#consider-opening-your-pull-request-as-draft
+-->
+
+**What this PR does / why we need it**:
+
+This PR adds the repository `repo_name` to the KubeVirt organization.
+
+_Checklist:_
```

Comment:
Please add
`- [ ] Link to email with announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/)`